### ISSUE TITLE
[Designer-Publish Adaptive Card Schema]: On activating 'Set due date' control focus does not lands to the 'View' control in forward navigation under 'Publish Adaptive Card Schema' card.

### ISSUE THREAD

**Issue by PankajBhojwani** [microsoft/AdaptiveCards#7156]
https://microsoft.visualstudio.com/OS/_workitems/edit/37777989

**Comment by PankajBhojwani** [comment]
Did some investigation into this issue, here's the problem:

In `card-elements.ts`, the function `expandShowCardAction` makes any actions _after_ the 'expanding' action not focusable, so that upon pressing `tab` focus goes directly to the expanded content. This is totally the intended behaviour when the card is rendered on the [samples page](https://adaptivecards.io/samples/ActivityUpdate.html)

However, the issue is that the [designer](https://adaptivecards.io/designer/) renders the _exact same card differently_ - instead of placing the actions side by side horizontally, it places the actions _vertically_, so it produces a seemingly strange behaviour where the action right below the 'expanding' action gets skipped over in the focus order. 

@paulcam206 @licanhua was there a reason why the designer page renders the action bar vertically instead of horizontally?

**Comment by PankajBhojwani** [comment]
@paulcam206 informed me that [host configs](https://docs.microsoft.com/en-us/adaptive-cards/rendering-cards/host-config#actionsconfig) are a thing, and that is what determines whether the actions are rendered vertically or horizontally.
